src/display/api.js

```diff
@@ -951,17 +951,60 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
     },
 
     /**
+     * @param {boolean} normalizeWhitespace - replaces all occurrences of
+     * whitespace with standard spaces (0x20). The default value is `false`.
+     * @param {boolean} disableCombineTextItems - do not attempt to combine
+     * same line {@link TextItem}'s. The default value is `false`.
+     * @return {ReadableStream} ReadableStream to read textContent chunks.
+     */
+    streamTextContent({ normalizeWhitespace = false,
+                        disableCombineTextItems = false, }) {
+      const TEXT_CONTENT_CHUNK_SIZE = 100;
+      return this.transport.messageHandler.sendWithStream('GetTextContent', {
+        pageIndex: this.pageNumber - 1,
+        normalizeWhitespace,
+        disableCombineTextItems,
```

Comment:
Please use the same validation as below, to ensure that what we're passing to the worker is sanitized!
```javascript
  normalizeWhitespace: normalizeWhitespace === true,
  combineTextItems: disableCombineTextItems !== true,
```